doc/Language/nativecall.pod6

```diff
@@ -186,6 +186,20 @@ When the signature of your native function needs a pointer to some native type
     sub my_version(int32 is rw, int32 is rw) is native('foo') { * }
     my_version(my int32 $major, my int32 $minor); # Pass a pointer to
 
+When you just want to use/pass a pointer using Pointer class you have to
+create the pointers when you declare them. Then you can do this without specifying
+what type it points to. Here is an example:
+
+    use NativeCall;
+    # C prototype is void create_object(void **object)
+    sub create_object(Pointer is rw) is native('foo') { * }
+    my Pointer $p = Pointer.new();
+    create_object($p); # pointer is set by create object
+
+The C<Pointer> class can be used with types as well e.g. Pointer<Str>..
```

Comment:
Please use again the pod marks for code in `Pointer[Str]`. There are also two periods at the end of that line. Please also note that `Pointer<Str>` is an error, which is not your fault, because it's elsewhere in the page (I'll fix it now), but an error anyway. I'd appreciate if you'd fix it.